src/_includes/soft_mod_cheat_sheet_cta_banner.html

```diff
@@ -0,0 +1,30 @@
+<div class="container section g-pt-200 g-pb-100">
+  <div class="soft-mod-in-page-banner">
+    <div class="soft-mod-cheat-sheet-cta-block-image-{{ site.baseurl | remove: '/' }}"></div>
+    <div class="soft-mod-cta-block__main">
```

Comment:
This classname suggests it is the `__main` element of a BEM style block called `soft-mod-cta-block`. Where is that block?